pkgs/development/python-modules/aiobotocore/default.nix

```
@@ -16,27 +17,22 @@
 
 buildPythonPackage rec {
   pname = "aiobotocore";
-  version = "2.8.0";
-  format = "setuptools";
+  version = "2.11.2";
+  pyproject = true;
 
   disabled = pythonOlder "3.8";
 
   src = fetchFromGitHub {
     owner = "aio-libs";
     repo = pname;
```

```suggestion
    repo = "aiobotocore";
```
https://github.com/NixOS/nixpkgs/issues/277994